functions/routes/index.js

```diff
@@ -5,15 +5,15 @@ const express = require("express");
 const onFleetService = require("../services/onfleet");
 const firebaseService = require("../services/firebase");
 const sendgridService = require("../services/sendgrid");
-
+const googleCloudService = require("../services/google-cloud");
 const router = express.Router({ mergeParams: true });
 
-router.get("/task/:id", async function(req, res) {
+router.get("/task/:id", async function (req, res) {
```

Comment:
please remove the whitespace between the function and the round brackets

Reply:
Done. Used yarn format.